test/ossl_shim/packeted_bio.h

```diff
@@ -32,4 +32,4 @@ timeval PacketedBioGetClock(const BIO *bio);
 bool PacketedBioAdvanceClock(BIO *bio);
 
 
-#endif  // HEADER_PACKETED_BIO
+#endif  // OPENSSL_PACKETED_BIO_H
```

Comment:
here as well.  I wonder why that does not raise any C90 warnings...

Reply:
ah, this is a C++ header, so the comment style is okay, I guess...

Reply:
I'll  fix them nevertheless.

Reply:
I took a closer look and you were right @bernd-edlinger: This header file is full of C++ style comments, it wouldn't make sense to fix this single location.